pkg/cidata/cidata.TEMPLATE.d/lima.env

```
@@ -46,3 +46,4 @@ LIMA_CIDATA_PLAIN=1
 {{- else}}
 LIMA_CIDATA_PLAIN=
 {{- end}}
+LIMA_TIMEZONE={{ .Timezone }}
```

For consistency with the other names it should use the `LIMA_CIDATA_` prefix:

```suggestion
LIMA_CIDATA_TIMEZONE={{ .Timezone }}
```

---

Updated in [97c113c](https://github.com/lima-vm/lima/pull/1931/commits/97c113cde70a6db042accad70143046000a44da2), didn't notice the `_CIDATA` part.